content/docs/guides/10-minute-tutorial.md

```diff
@@ -1413,18 +1425,18 @@ Running hellocucumber.RunCucumberTest
 Feature: Is it Friday yet?
   Everybody wants to know when it's Friday
 
-  Scenario: Friday is Friday           # hellocucumber/is_it_friday_yet.feature:4
-    Given today is Friday              # Stepdefs.today_is_Sunday()
-    When I ask whether it's Friday yet # Stepdefs.i_ask_whether_it_s_Friday_yet()
-    Then I should be told "TGIF"       # Stepdefs.i_should_be_told(String)
-
   Scenario: Sunday isn't Friday        # hellocucumber/is_it_friday_yet.feature:4
     Given today is Sunday              # Stepdefs.today_is_Sunday()
     When I ask whether it's Friday yet # Stepdefs.i_ask_whether_it_s_Friday_yet()
     Then I should be told "Nope"       # Stepdefs.i_should_be_told(String)
 
-2 scenarios (2 passed)
-6 steps (6 passed)
+  Scenario: Friday is Friday           # hellocucumber/is_it_friday_yet.feature:9
```

Comment:
Not sure why the order of the scenarios has changed. But the new order here makes sense. :+1:

Reply:
We started ordering events in v4.